lib/change_stream.js

```diff
@@ -309,7 +309,7 @@ var buildChangeStreamAggregationCommand = function(self) {
   if (resumeToken) {
     changeStreamStageOptions.resumeAfter = resumeToken;
   }
-
+  
```

Comment:
It looks like this whitespace is causing the linter to fail. Can we remove it?

Reply:
Oops, fire and forget push last night.  Should be fixed now.  FYI, the sharded and unified test suites are failing but errors seem unlikely to be related to this change.  Thanks @daprahamian !